crates/stc_ts_file_analyzer/tests/pass/exprs/member/indexSignature/mutliple/1.swc-stderr

```diff
@@ -72,7 +72,7 @@ warning: Type
 12 |     var e = r2['1'];
    |             ^^^^^^^
    |
-   = note: any
+   = note: Object
```

Comment:
These two are also closer to TS actual value (Ts infers it as type T whereas this is type Object)

I guess typescript attempts to parse '1' to a number first